spec/user_spec.rb

```diff
@@ -0,0 +1,12 @@
+describe "User" do
+
+  describe "#authenticate" do
```

Comment:
Is this an instance method? I think it might be a `.authenticate` method instead